var/spack/repos/builtin/packages/singularityce/package.py

```diff
@@ -36,11 +44,11 @@ def gopath(self):
 
     @property
     def sylabs_gopath_dir(self):
-        return join_path(self.gopath, 'src/github.com/sylabs/')
+        return join_path(self.gopath, 'src', 'github.com', self.singularity_org)
```

Comment:
Maybe rename `sylabs_gopath_dir` since it's in the base package :grimacing: ?

Reply:
Also, should we do something about `singularity`?

Reply:
I was tempted to just remove that property and merge it with `singularity_gopath_dir`, as it's not used anywhere else. But I can also rename it, any suggestions? Or am I misunderstanding you?

W.r.t. `singularity`, it seems that the Github page states that it is EOL, in which case it should be deprecated. OTOH, there is a fairly recent release there, too.

Reply:
Removing the property is fine with me.